testing/php7-yaml/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=php7-yaml
+_pkgreal=yaml
+pkgver=2.0.0
+pkgrel=0
+_phpver=${pkgname#php}
+_phpver=${_phpver%%-*}
+pkgdesc="PHP${_phpver}"
+url="http://bd808.com/pecl-file_formats-yaml/"
+arch="all"
+license="MIT"
+makedepends="php${_phpver}-dev autoconf yaml-dev"
+source="${pkgname}-${pkgver}.tgz::http://pecl.php.net/get/${_pkgreal}-${pkgver}.tgz"
+builddir="${srcdir}/${_pkgreal}-${pkgver}/"
+options="!check"
```

Comment:
Yes, tests fails.

I've submitted an official report.

For the record, here is the `make test` output:

```
=====================================================================
PHP         : /usr/bin/php7 
PHP_SAPI    : cli
PHP_VERSION : 7.1.6
ZEND_VERSION: 3.1.0
PHP_OS      : Linux - Linux 19eb4b5e81e5 4.10.0-22-generic #24-Ubuntu SMP Mon May 22 17:43:20 UTC 2017 x86_64
INI actual  : /work/testing/php7-yaml/src/pecl-file_formats-yaml-php7/tmp-php.ini
More .INIs  :   
CWD         : /work/testing/php7-yaml/src/pecl-file_formats-yaml-php7
Extra dirs  : 
VALGRIND    : Not used
=====================================================================
TIME START 2017-06-13 18:30:06
=====================================================================
PASS Test PECL bug #21995 [tests/bug_21995.phpt] 
PASS Test PECL bug #59860 [tests/bug_59860.phpt] 
PASS Test PECL bug #260628 [tests/bug_60628.phpt] 
PASS Test PECL bug #61770 [tests/bug_61770.phpt] 
PASS Test PECL bug #61923 [tests/bug_61923.phpt] 
PASS Test PECL bug #64019 [tests/bug_64019.phpt] 
PASS Test PECL bug #64694 [tests/bug_64694.phpt] 
PASS Test PECL bug #69465 [tests/bug_69465.phpt] 
PASS Test PECL bug #69616 [tests/bug_69616.phpt] 
PASS Test PECL bug #69617 [tests/bug_69617.phpt] 
PASS Test PECL bug #72204 [tests/bug_72204.phpt] 
PASS Test PECL strange alias bug [tests/bug_parsing_alias.phpt] 
FAIL yaml - emit -> parse roundtrip [tests/yaml_001.phpt] 
PASS yaml - emit -> parse roundtrip with datetime [tests/yaml_002.phpt] 
PASS yaml_emit - scalars [tests/yaml_emit_001.phpt] 
PASS yaml_emit - sequences [tests/yaml_emit_002.phpt] 
PASS yaml_emit - mappings [tests/yaml_emit_003.phpt] 
PASS yaml_emit - datetime [tests/yaml_emit_004.phpt] 
PASS yaml_emit - serializable [tests/yaml_emit_005.phpt] 
PASS yaml_emit - ini settings [tests/yaml_emit_006.phpt] 
PASS yaml_emit - recursive structures [tests/yaml_emit_007.phpt] 
PASS yaml_emit - custom tags [tests/yaml_emit_008.phpt] 
PASS yaml_emit - custom tags with closures [tests/yaml_emit_009.phpt] 
PASS Test Github pull request #1 [tests/yaml_emit_file_basic.phpt] 
FAIL yaml_parse - general [tests/yaml_parse_001.phpt] 
PASS yaml_parse - multiple documents [tests/yaml_parse_002.phpt] 
PASS yaml_parse - parse errors [tests/yaml_parse_003.phpt] 
PASS yaml_parse - syck bug #11585 [tests/yaml_parse_004.phpt] 
PASS yaml_parse - syck bug #12656 [tests/yaml_parse_005.phpt] 
PASS yaml_parse - syck bug #14384 [tests/yaml_parse_006.phpt] 
PASS yaml_parse - serializable [tests/yaml_parse_007.phpt] 
PASS yaml_parse - callbacks [tests/yaml_parse_008.phpt] 
FAIL yaml_parse_file - general [tests/yaml_parse_file_001.phpt] 
PASS yaml_parse_file - error cases [tests/yaml_parse_file_002.phpt] 
PASS Yaml 1.1 Spec - binary [tests/yaml_parse_spec_binary.phpt] 
PASS Yaml 1.1 Spec - bool [tests/yaml_parse_spec_bool.phpt] 
FAIL Yaml 1.1 Spec - float [tests/yaml_parse_spec_float.phpt] 
PASS Yaml 1.1 Spec - int [tests/yaml_parse_spec_int.phpt] 
PASS Yaml 1.1 Spec - map [tests/yaml_parse_spec_map.phpt] 
PASS Yaml 1.1 Spec - merge [tests/yaml_parse_spec_merge.phpt] 
PASS Yaml 1.1 Spec - null [tests/yaml_parse_spec_null.phpt] 
PASS Yaml 1.1 Spec - omap [tests/yaml_parse_spec_omap.phpt] 
PASS Yaml 1.1 Spec - pairs [tests/yaml_parse_spec_pairs.phpt] 
PASS Yaml 1.1 Spec - seq [tests/yaml_parse_spec_seq.phpt] 
PASS Yaml 1.1 Spec - set [tests/yaml_parse_spec_set.phpt] 
PASS Yaml 1.1 Spec - str [tests/yaml_parse_spec_str.phpt] 
PASS Yaml 1.1 Spec - timestamp [tests/yaml_parse_spec_timestamp.phpt] 
PASS Yaml YtsBasicTests - Simple Sequence [tests/yaml_parse_wiki_YtsBasicTests_001.phpt] 
PASS Yaml YtsBasicTests - Nested Sequences [tests/yaml_parse_wiki_YtsBasicTests_002.phpt] 
PASS Yaml YtsBasicTests - Mixed Sequences [tests/yaml_parse_wiki_YtsBasicTests_003.phpt] 
PASS Yaml YtsBasicTests - Deeply Nested Sequences [tests/yaml_parse_wiki_YtsBasicTests_004.phpt] 
PASS Yaml YtsBasicTests - Simple Mapping [tests/yaml_parse_wiki_YtsBasicTests_005.phpt] 
PASS Yaml YtsBasicTests - Sequence in a Mapping [tests/yaml_parse_wiki_YtsBasicTests_006.phpt] 
PASS Yaml YtsBasicTests - Nested Mappings [tests/yaml_parse_wiki_YtsBasicTests_007.phpt] 
PASS Yaml YtsBasicTests - Mixed Mapping [tests/yaml_parse_wiki_YtsBasicTests_008.phpt] 
PASS Yaml YtsBasicTests - Sequence-Mapping Shortcut [tests/yaml_parse_wiki_YtsBasicTests_009.phpt] 
PASS Yaml YtsBlockMapping - One Element Mapping [tests/yaml_parse_wiki_YtsBlockMapping_001.phpt] 
PASS Yaml YtsBlockMapping - Multi Element Mapping [tests/yaml_parse_wiki_YtsBlockMapping_002.phpt] 
PASS Yaml YtsBlockMapping - Values aligned [tests/yaml_parse_wiki_YtsBlockMapping_003.phpt] 
PASS Yaml YtsBlockMapping - Colons aligned [tests/yaml_parse_wiki_YtsBlockMapping_004.phpt] 
PASS Yaml YtsMapInSeq - Single key [tests/yaml_parse_wiki_YtsMapInSeq_001.phpt] 
PASS Yaml YtsMapInSeq - Multiple keys [tests/yaml_parse_wiki_YtsMapInSeq_002.phpt] 
PASS Yaml YtsMapInSeq - Strange keys [tests/yaml_parse_wiki_YtsMapInSeq_003.phpt] 
PASS Yaml YtsStrangeKeys - Quoted line noise key [tests/yaml_parse_wiki_YtsStrangeKeys_001.phpt] 
PASS Yaml YtsStrangeKeys - Unquoted line noise key [tests/yaml_parse_wiki_YtsStrangeKeys_002.phpt] 
=====================================================================
TIME END 2017-06-13 18:30:07

=====================================================================
TEST RESULT SUMMARY
---------------------------------------------------------------------
Exts skipped    :    0
Exts tested     :   11
---------------------------------------------------------------------

Number of tests :   65                65
Tests skipped   :    0 (  0.0%) --------
Tests warned    :    0 (  0.0%) (  0.0%)
Tests failed    :    4 (  6.2%) (  6.2%)
Expected fail   :    0 (  0.0%) (  0.0%)
Tests passed    :   61 ( 93.8%) ( 93.8%)
---------------------------------------------------------------------
Time taken      :    1 seconds
=====================================================================

=====================================================================
FAILED TEST SUMMARY
---------------------------------------------------------------------
yaml - emit -> parse roundtrip [tests/yaml_001.phpt]
yaml_parse - general [tests/yaml_parse_001.phpt]
yaml_parse_file - general [tests/yaml_parse_file_001.phpt]
Yaml 1.1 Spec - float [tests/yaml_parse_spec_float.phpt]
=====================================================================
```